common/lib/xmodule/xmodule/tabs.py

```diff
@@ -121,14 +133,8 @@ def __setitem__(self, key, value):
 
         Note: the 'type' member can be 'get', but not 'set'.
```

Comment:
Thanks for the reminder. I've pushed up logic to have a list of read-only attributes.